app/controllers/concerns/list_description_helper.rb

```diff
@@ -1,11 +1,13 @@
 module ListDescriptionHelper
   # Helper method to create hash for a ListDescriptionComponentSerializer item
-  def create_description_list_item(term, descriptions)
+  def create_description_list_item(term: nil, descriptions: nil)
     return if descriptions.all?(&:blank?)
 
     {}.tap do |hash|
-      hash[:term] = { content: term }
-      hash[:description] = descriptions.map { |description| { content: description } }
+      hash[:term] = { content: term } if term.is_a? String
+      hash[:term] = term if term.is_a? Hash
+      hash[:description] = descriptions.map { |description| { content: description } } if descriptions.any? { |item| item.is_a? String }
+      hash[:description] = descriptions if descriptions.any? { |item| item.is_a? Hash }
```

Comment:
This will cause weirdness if one is a string and another is a hash etc. Consider using `#all`, or creating a combined descriptions object above the tap

Reply:
With the above, this becomes:

```ruby 
hash[:description] = descriptions
```

Reply:
🍰

Reply:
Test written and updates made